test/plugin-prepare.ts

```diff
@@ -32,20 +32,6 @@ describe("Plugin preparation", () => {
 		assert.deepEqual({}, pluginPrepare.preparedDependencies);
 	});
 
-	it("skips prepare if every plugin prepared", async () => {
```

Comment:
does it mean that now we'll never skip prepare even if all plugins are okay?

Reply:
no, if the JS part of the plugins is prepared we will skill prepare, but we will always check for changes in the native part of a plugin for changes that need rebuilding.